File: C/array_serch.c
#include <stdio.h>
#include <stdlib.h>
int main(void)
{
    // declare array and limit
    int value[100];
    // declare variable
    int i, limit, serchkey, flag = 0;
    printf("Enter array Limit");
    scanf("%d", &limit);
    printf("Enter values");
    // checking limt
    for (i = 0; i < limit; i++)
    {
        // adding values array & position
        scanf("%d", &value[i]);
    }
    printf("Please enter serch key");
    scanf("%d", &serchkey);
    // re run arrya for serch
    for (i = 0; i < limit; i++)
    {
        // checking serch key
        if (serchkey == value[i])
        {
            // its true flag 1 and exit loop
            flag = 1;
            break;
        }
    }
    if (flag == 1)
    {
        // i= loop exiting position, user comfirt + 1
        printf("value founded \n at position : %d", i + 1);
    }
    else
    {
        printf("Value Not found!!");
    }
}

File: JAVA/Array/Array_serch.java
import java.util.Scanner;

public class Array_serch {
    public static void main(String[] args) {
        // input
        Scanner s = new Scanner(System.in);
        System.out.println("Enter limit");
        // get limit
        int limit = s.nextInt();
        System.out.println("Enter Values");
        // declare array
        int[] values = new int[100];
        // save value at position
        for (int i = 0; i < limit; i++) {
            values[i] = s.nextInt();
        }
        // get serch key
        System.out.println("Enter Serch Key");
        int serchkey = s.nextInt();
        // set a key & valu 0
        int flag = 0;
        // serch
        for (int i = 0; i < limit; i++) {
            if (serchkey == values[i]) {
                flag = 1;
                break;
            }

        }
        // print
        if (flag == 1) {
            System.out.println("Value founded");
        } else {
            System.out.println("Value Not Founded");
        }
    }
}
